### ISSUE TITLE
can't create a new map

### ISSUE THREAD

**Issue by rockBreaker** [exodus4d/pathfinder#82]
Hi i signed up today and basically i cant create a new map.
I keep getting server error 500's too. 
This means whenever i click something else, I see that popup every 6 seconds.

Any idea why this is happening?

**Comment by exodus4d** [comment]
sounds interesting. I have never heard about anything linke that. Are you using my installation or do you have your own installation running? If it is mine, I´ll check the log files for any errors. Have you recently changed anything like API Keys, switched Corp/Ally? How many maps do you have/had open?

**Comment by rockBreaker** [comment]
I was using your installation. Username Zordak. 
I have recently changed Corp. I tried deleting my account as well then creating a  new one and no joy.
I'll try creating another account if you like?

**Comment by Tupsi** [comment]
this might be related to problems I got from my corp mates reported. It is not related to creating a new map, but to creating object on the map current map in generel, resulting in a 500 server error. For instance, someone jumps a wh and gets the error. The map shows the new system after a few moments, but NOT the connection wh (and yes, auto mapping is on). He then can draw the connecting wh manually just fine.
"I got a 500: savesystem 'error internal server error' jumping from the static to the c5.3 and it didnt track me into the 5.3"
